### ISSUE TITLE
Question：About the behaviour of solanum in new profile

### ISSUE THREAD

**Issue by solitary-sails** [hearth1an/hearth1an.ChristmasStory#47]
The behaviour of Solanum in base game won't be touch off .So how Hatchling know her name when ask Hal to write sentence.

**Comment by hearth1an** [comment]
Let's be honest, mod players have already beaten the base game and the DLC, perhaps multiple times. I don't want players to lie to themselves and say that they don't know who Solanum is or how to get to the Prisoner or QM/ATP, etc.

Moreover, it would be a very silly experience if you visit Solanum as usual, talk to her as usual, but only get invited on the next loop. What's the point?

**Comment by solitary-sails** [comment]
I understand that players know those knowledge. So in this case ,why you suggest player play in a new profile in mod popup text. Just make it easier to let player talk with Slate?

**Comment by hearth1an** [comment]
1. Player will go for launch codes and will meet  and talk to Hal and also could talk to other hearthians.
2. Player will keep their main profile untouchable

**Comment by solitary-sails** [comment]
Thanks for answering.  Although this makes me feel weird, I can understand your good intentions.

By the way, I have played a mod before, but unfortunately it's so long ago that I can't remember exactly what it was.  In that mod, after entering the game, the player will notice that the launch code has been changed, which makes the player have to go to the observatory again.This can also achieve the first goal you mentioned, but unfortunately it cannot prevent the player's main profile from being touched.
